### ISSUE TITLE
Add license

### ISSUE THREAD

**Issue by Kaushik080** [areweguiyet/areweguiyet#133]
Add license for the repository

**Comment by Kaushik080** [comment]
I would like to work on this, Please assign me

**Comment by madsmtm** [comment]
To do this, we'd need all of the [contributors](https://github.com/areweguiyet/areweguiyet/graphs/contributors) to sign off on this, so I really doubt that's gonna happen.

**Comment by ccoenen** [comment]
With the absolute most work coming from 5 people (@shivshank, @dxenonb, @madsmtm, @dhardy and @pythoneer according to https://github.com/areweguiyet/areweguiyet/graphs/contributors ) this should not be _too_ hard. Everyone else has contributed few commits and/or few lines. So, if they do not consent to a license change of their contributions, it is probably possible to recreate the work.

Also, some of the contributions will have been collection of factual statements, where it is doubtful if this even creates a copyrightable work.

**Comment by madsmtm** [comment]
Since you've already notified them, then sure, let's try:

---

We're going to change the license for the work in this repository to the trio-license `Zlib OR Apache-2.0 OR MIT`, if you consent to this, please state so in a comment (preferred), or react to this message with a 🚀.

**Comment by dhardy** [comment]
I consent to a licence-change to `Zlib OR Apache-2.0 OR MIT`.

Not sure about consent-via-emoji :laughing:

**Comment by dxenonb** [comment]
I also approve a license change to Zlib OR Apache-2.0 OR MIT (shivshank is me... so I also speak for them; old username).

Regarding the legality, I don't think there'd be any issue saying all future contributions are under that license; and most of the stuff we display is also licensed under some OSS/FOSS license. And then for the handful of strings that are original content, it would require that person to take someone to court... over someone reproducing content from a website made to help people find libraries they can use. 😅

I do apologize for the oversight in not providing one initially! It's a good idea.

**Comment by pythoneer** [comment]
I hereby consent to the proposed license change to `Zlib OR Apache-2.0 OR MIT`

**Comment by madsmtm** [comment]
Yeah, the consent-via-emoji was maybe ill conceived... Anyhow, I've added the license texts in https://github.com/areweguiyet/areweguiyet/pull/137, thanks all for consenting!
